deploy/helm/flightctl/templates/flightctl-default-user-roles.yaml

```diff
@@ -0,0 +1,90 @@
+{{ if eq .Values.global.target "acm" }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-admin
+  namespace: {{ .Release.Namespace }}
+rules:
+  - verbs:
+      - '*'
+    apiGroups:
+      - flightctl.io
+    resources:
+      - '*'
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-viewer
+  namespace: {{ .Release.Namespace }}
+rules:
+  - verbs:
+      - get
+      - list
+    apiGroups:
+      - flightctl.io
+    resources:
+      - devices
+      - fleets
+      - resourcesyncs
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-operator
+  namespace: {{ .Release.Namespace }}
+rules:
+  - verbs:
+      - get
+      - list
+      - create
+      - delete
+      - update
+      - patch
+    apiGroups:
+      - flightctl.io
+    resources:
+      - devices
+      - fleets
+      - resourcesyncs
+  - verbs:
+      - get
+      - list
+    apiGroups:
+      - flightctl.io
+    resources:
+      - repositories
+
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: flightctl-installer
+  namespace: {{ .Release.Namespace }}
+rules:
+  - verbs:
+      - get
+      - list
+      - create
+      - delete
+      - update
+      - patch
+      - post
+    apiGroups:
+      - flightctl.io
+    resources:
+      - enrollmentrequests
+      - enrollmentrequests/approval
+      - certificatesigningrequests
+      - certificatesigningrequests/approval
```

Comment:
I think they need to create/list/get CSRs for the enrollment-certs (though I have a headache that they'll be able to access also get certs they didn't request themselves, esp. agent certs this way), but the enrollment certs should be auto-approved, so they shouldn't need/have that permission, I think.